From 058bc7a903672a01981450a8c21ec03b92498aff Mon Sep 17 00:00:00 2001
From: Benjamin Franzke <benjaminfranzke@googlemail.com>
Date: Thu, 19 Feb 2015 13:10:18 +0100
Subject: [PATCH] man: add newlines to the pull-raw example in machinectl(1)

They were removed in commit 798d3a52 ("Reindent man pages to 2ch").

(cherry picked from commit ac92ced5bb41def1d90f871d6c8cfec2b03c0c7d)
---
 man/machinectl.xml | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/man/machinectl.xml b/man/machinectl.xml
index 9b07af4226..640cb8b7d6 100644
--- a/man/machinectl.xml
+++ b/man/machinectl.xml
@@ -715,11 +715,12 @@
       <title>Download a Fedora image, set a root password in it, start
       it as service</title>
 
-      <programlisting># machinectl pull-raw --verify=no
-      http://ftp.halifax.rwth-aachen.de/fedora/linux/releases/21/Cloud/Images/x86_64/Fedora-Cloud-Base-20141203-21.x86_64.raw.xz
-      # systemd-nspawn -M Fedora-Cloud-Base-20141203-21 # passwd #
-      exit # machinectl start Fedora-Cloud-Base-20141203-21 #
-      machinectl login Fedora-Cloud-Base-20141203-21</programlisting>
+      <programlisting># machinectl pull-raw --verify=no http://ftp.halifax.rwth-aachen.de/fedora/linux/releases/21/Cloud/Images/x86_64/Fedora-Cloud-Base-20141203-21.x86_64.raw.xz
+# systemd-nspawn -M Fedora-Cloud-Base-20141203-21
+# passwd
+# exit
+# machinectl start Fedora-Cloud-Base-20141203-21
+# machinectl login Fedora-Cloud-Base-20141203-21</programlisting>
 
       <para>This downloads the specified <filename>.raw</filename>
       image with verification disabled. Then a shell is opened in it
